### PR TITLE
Action header - Add AriaLabel Text

### PR DESCRIPTION
**Pull request by jmsv6d**
### Summary
Adding aria-label text for the icons on the buttons in the action header. 

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-clinical-action-header/lib/ActionHeader.js [hunk 1]
```diff
@@ -104,6 +114,11 @@ var propTypes = {
   onPrevious: _propTypes2.default.func,
 
   /**
+   * Locale for i18n of accessability text.
+   **/
+  locale: _propTypes2.default.string,
```
Comment (Matt-Butler) [inline packages/terra-clinical-action-header/lib/ActionHeader.js]: Should setting locale be at the application level?

Comment (Matt-Butler) [inline packages/terra-clinical-action-header/lib/ActionHeader.js]: @windse7en @ryanthemanuel

File: packages/terra-clinical-action-header/package.json [hunk 2]
```diff
@@ -23,18 +23,21 @@
...
   "homepage": "https://github.com/cerner/terra-clinical#readme",
   "devDependencies": {
-    "terra-toolkit": "^0.x"
+    "terra-toolkit": "^0.x",
+    "terra-i18n-plugin": "^0.x"
   },
   "peerDependencies": {
     "react": "15.4.2",
     "react-dom": "15.4.2",
+    "react-intl": "^2.3.0",
```
Comment (emilyrohrbough) [inline packages/terra-clinical-action-header/package.json]: We need this package?

File: packages/terra-clinical-action-header/src/ActionHeader.jsx [hunk 3]
```diff
@@ -69,108 +75,148 @@ const defaultProps = {
   onNext: null,
   onPrevious: null,
   children: null,
+  locale: 'en-US',
```
Comment (StephenEsser) [inline packages/terra-clinical-action-header/src/ActionHeader.jsx]: I'd talk to some of the i18n people, but I thought I remembered someone saying we shouldn't default locale's in one of our meetings

Comment (emilyrohrbough) [inline packages/terra-clinical-action-header/src/ActionHeader.jsx]: Is the locale handled by the release of `terra-base` v0.5.0?

**Comment by mjhenkes** [comment]
@jmsv6d are there changes you still need to make on this PR?

File: packages/terra-clinical-action-header/lib/ActionHeader.js [hunk 4]
```diff
@@ -0,0 +1,243 @@
+'use strict';
```
Comment (bjankord) [inline packages/terra-clinical-action-header/lib/ActionHeader.js]: The `lib` directory is now [ignored from version control](https://github.com/cerner/terra-clinical/blob/master/.gitignore#L53) and [generated via the compile script after running npm install and when publishing an npm package](https://github.com/cerner/terra-clinical/blob/master/package.json#L55-L56).

This means we can delete this file.

Comment (jmsv6d) [inline packages/terra-clinical-action-header/lib/ActionHeader.js]: Deleted file

File: .gitignore [hunk 5]
```diff
@@ -51,3 +51,4 @@ reports
 screenshots
 target
 lib
+**/aggregated-translations/*.*
```
Comment (ryanthemanuel) [inline .gitignore]: If we're adding this to .gitignore, we should also delete the folder itself.

File: packages/terra-clinical-action-header/src/ActionHeader.jsx [hunk 6]
```diff
@@ -71,6 +76,15 @@ const defaultProps = {
   children: null,
 };
 
+const contextTypes = {
```
Comment (emilyrohrbough) [inline packages/terra-clinical-action-header/src/ActionHeader.jsx]: I am still curious, with adding `contextTypes`, do we still need to pass in the `locale` prop.

Comment (jmsv6d) [inline packages/terra-clinical-action-header/src/ActionHeader.jsx]: Ooops, no. Removed the locale prop.

**Pull request merged**